examples/compose/compose.yml

```diff
@@ -2,12 +2,12 @@ name: nextcloud
 
 services:
   db:
-    image: mariadb:10.5
+    image: mariadb:10
     container_name: nextcloud_db
     volumes:
       - "db:/var/lib/mysql"
     environment:
-      - "MYSQL_ALLOW_EMPTY_PASSWORD=yes"
+      - "MARIADB_RANDOM_ROOT_PASSWORD=yes"
```

Comment:
These changes don't seem related to your changes but ones from head: https://github.com/crazy-max/docker-nextcloud/commit/f289a92080d1c37e1ae47e5566058f07a9b150b9

Can you do a proper rebase please?